### ISSUE TITLE
Broken links in track description

### ISSUE THREAD

**Issue by fanburst** [fanburst/fanburst#4]
Links in track and album descriptions are not clickable.

**Comment by fanburst** [comment]
This is now fixed.

**Comment by BitByteMC** [comment]
@fanburst I know this issue is closed, but is there any way we could make links work without having to use http:// in the link?

**Comment by fanburst** [comment]
Yeah it should definitely work that way, I'll reopen this and look into it this week.

**Comment by AtriusEDM** [comment]
Great, thanks for the fast reply! Also sorry if I'm spamming the repo with issues, just trying to make sure everything is noticed. :)

**Comment by fanburst** [comment]
No worries at all, it's really helpful so keep it coming :D
